### PR TITLE
Exclude known issues on jdk8

### PR DESCRIPTION
**Pull request by sophia-guo**
https://github.com/adoptium/aqa-tests/issues/4966

### CODE REVIEW

File: openjdk/excludes/ProblemList_openjdk8.txt [hunk 1]
```diff
@@ -146,7 +146,7 @@ sun/management/jmxremote/bootstrap/RmiSslNoKeyStoreTest.sh https://github.com/ad
 # alpine-linux https://github.com/adoptium/aqa-tests/issues/3238
 sun/management/jmxremote/bootstrap/CustomLauncherTest.java https://bugs.openjdk.java.net/browse/JDK-8031420 linux-all
 com/sun/management/OperatingSystemMXBean/TestTotalSwap.java https://bugs.openjdk.java.net/browse/JDK-8255263 linux-all
-
+sun/management/jmxremote/startstop/JMXStartStopTest.java https://github.com/adoptium/aqa-tests/issues/3948 generic-all
```
Comment (smlambert) [inline openjdk/excludes/ProblemList_openjdk8.txt:149]: I will remove this one from my [PR](https://github.com/adoptium/aqa-tests/pull/4965/), noting that I used  https://bugs.openjdk.org/browse/JDK-8274337 as I believe it to be the same issue (though currently only reported against Mac).

**Review by smlambert — APPROVED** [review]
LGTM, though we can reference the upstream issue for JMXStartStopTest.

**Review by karianna — APPROVED** [review]
same as shelly

File: openjdk/excludes/ProblemList_openjdk8.txt (hunk 1 above)
Comment (sophia-guo) [reply openjdk/excludes/ProblemList_openjdk8.txt]: Updated.

**Pull request merged**